src/components/molecules/InboxItem.vue

```diff
@@ -1,34 +1,69 @@
 <template>
-  <div :class="[selected ? 'bg-blue-selected':'', 'flex items-center w-full h-16 md:h-20 border-2 border-transparent focus:border-black hover:bg-gray-100 cursor-pointer']">
+  <div
+    :class="[
+      selected ? 'bg-blue-selected' : '',
+      'flex items-center w-full h-16 md:h-20 border-2 border-transparent focus:border-black hover:bg-gray-100 cursor-pointer',
+    ]"
+  >
     <div class="p-4">
-      <p class="bg-blue-light rounded-full flex items-center justify-center font-heading w-10 h-10 md:w-12 md:h-12 text-3xl">
-        {{icon}}
+      <p
+        class="
+          bg-blue-light
+          rounded-full
+          flex
+          items-center
+          justify-center
+          font-heading
+          w-10
+          h-10
+          md:w-12 md:h-12
+          text-3xl
+        "
+      >
+        {{ icon }}
       </p>
     </div>
     <div class="flex-auto truncate">
-        <p :class="[!dayRead ? 'font-bold':'', 'font-body md:text-lg truncate overflow-ellipsis text-gray-dark']">Virtual Concierge</p>
-        <p class="font-body font-light text-sm md:text-base truncate overflow-ellipsis text-gray-dark">Hi Mary, I'm your Virtual Con...</p>
+      <p
```

Comment:
I find this hard to read, can you make it so that it doesn't break up the expression so much?

Reply:
I'd have to disagree, I find this much cleaner than the single line. These are prettier's default rules but if the teams wants we can come up with a formatting that fits all of our needs better

Reply:
I was thinking more like how it was done in the benefits finder,